pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/new/index.md

```diff
@@ -60,6 +60,8 @@ enterprise: false
 
 1.  Generate the Kubernetes cluster objects:
 
+    <p class="message--note"><strong>NOTE: </strong>To increase <a href="https://docs.docker.com/docker-hub/download-rate-limit/">Dockerhub's rate limit</a> you may use your Dockerhub credentials when the cluster is being created by setting additional flags <code>--registry-mirror-url=https://registry-1.docker.io --registry-mirror-username= --registry-mirror-password=</code> when creating the cluster with <code>dkp create cluster</code></p>
```

Comment:
```suggestion
        <p class="message--note"><strong>NOTE: </strong>To increase <a href="https://docs.docker.com/docker-hub/download-rate-limit/">Dockerhub's rate limit</a> use your Dockerhub credentials when creating the cluster, by setting flags <code>--registry-mirror-url=https://registry-1.docker.io --registry-mirror-username= --registry-mirror-password=</code> then running <code>dkp create cluster</code></p>
```

Reply:
@cmastr these flags are to be used for `dkp create cluster` command, "then running" sounds like these flags are run first and then `dkp create cluster` is run. Can you please update the suggestion with that in mind (if needed) for this one and I will copy it in the other occurrences.

Reply:
Also why do we need the extra spaces, the formatting looks fine as is
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/4969231/167216891-ae76a84b-a18d-45f1-a5b6-8085fc660abb.png">